streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignor.java

```diff
@@ -1055,9 +1060,10 @@ protected boolean maybeUpdateSubscriptionVersion(final int receivedAssignmentMet
                 log.info(
                     "Sent a version {} subscription and got version {} assignment back (successful version probing). "
                         +
-                        "Downgrade subscription metadata to commonly supported version and trigger new rebalance.",
+                        "Downgrade subscription metadata to commonly supported version {} and trigger new rebalance.",
```

Comment:
nitpick: let's just use `downgrade` than `downgrading` in the log4j?

Reply:
Ack, will replace all with `downgrade`